### PR TITLE
`addPerson` returns `ref` instead of `personId`

### PR DESCRIPTION
**Pull request by sirLisko**
**What**  
- Removed `personId` from `components/Steps/Summary`, used `ref` instead, so it's not bound to any page logic
- removed `ref` from `addCase` as doesn't make sense showing the id of the mongodb record to the user

**How to test it**
Create person should work as before
Create new case should work as before but the ref shouldn't be visible anymore

Screenshot 2021-02-23 at 17 27 06

### CODE REVIEW

File: utils/api/residents.js [hunk 1]
```diff
@@ -12,5 +12,5 @@ export const useResident = (personId) => useSWR(`/api/residents/${personId}`);
 
 export const addResident = async (formData) => {
   const { data } = await axios.post(`/api/residents`, formData);
-  return data;
+  return { ref: data?.personId, data };
 };
```
Comment (naomigreen) [inline utils/api/residents.js:16]: We're not getting the personId back as a ref, resulting in undefined on the create person confirmation page

**Review by naomigreen — CHANGES_REQUESTED** [review]

File: utils/api/residents.js (hunk 1 above)
Comment (sirLisko) [reply utils/api/residents.js:16]: ehm... did you try it?

Comment (sirLisko) [reply utils/api/residents.js:16]: this is actually returning the `personId` as `ref` 👇 
Screenshot 2021-02-26 at 15 01 27

**Review by naomigreen — APPROVED** [review]
Working now

**Comment by sirLisko** [comment]
It was working even before O_o

**Pull request merged**